src/ragapp/entrypoint.sh

```diff
@@ -23,5 +23,13 @@ else
     echo "Updated static files successfully!"
 fi
 
+# Copy default config to mounted volume if it is empty
+if [[ -z "$(ls -A /app/state/manager/config)" ]]; then
```

Comment:
is that a dependency from ragapp to the manager?

Reply:
Sorry, it was my mistake when mounting with a wrong path to RAGapp, it should be `/app/config`. It not actually a dependency from manager - just handle for the case that the app config is mounted with a empty folder it's also helpful for the case user want to mount config to update it from the host.